test/tests/functional/pbs_checkpoint.py

```diff
@@ -44,40 +44,67 @@ class TestCheckpoint(TestFunctional):
     """
     This test suite targets Checkpoint functionality.
     """
+    abort_file = ''
+    cu = CrayUtils()
 
-    def test_checkpoint_abort_with_preemept(self):
+    def test_checkpoint_abort_with_preempt(self):
+        """
+        This test verifies that checkpoint_abort works as expected by
+        1. setting preempt_order in sched_config to 'C'.
+        2. submitting a job to the default queue.
+        3. submitting a job to the express queue.
+        4. confirming that the first job was checkpointed.
+        """
         abort_script = """#!/bin/bash
 kill $1
 exit 0
 """
-        abort_file = self.du.create_temp_file(body=abort_script)
-        self.du.chmod(path=abort_file, mode=0755)
-        c = {'$action': 'checkpoint_abort 30 !' + abort_file + ' %sid'}
+        self.abort_file = self.du.create_temp_file(body=abort_script)
+        self.du.chmod(path=self.abort_file, mode=0755)
+        c = {'$action': 'checkpoint_abort 30 !' + self.abort_file + ' %sid'}
         self.mom.add_config(c)
+
         self.scheduler.set_sched_config({'preempt_order': 'C'})
+
         a = {'queue_type': 'e',
              'started': 'True',
              'enabled': 'True',
              'Priority': 200}
         self.server.manager(MGR_CMD_CREATE, QUEUE, a, "expressq")
+
         self.platform = self.du.get_platform()
         if self.platform != 'cray' and self.platform != 'craysim':
             node_attrs = self.server.status(NODE, id=self.server.shortname)[0]
```

Comment:
since the you changed how you form the job request, node_attrs is now no longer used.  Please remove it.